### ISSUE TITLE
Solution to AttributeError: 'Arrow3D' object has no attribute 'do_3d_projection'

### ISSUE THREAD

**Issue by diplodocuslongus** [phausamann/rigid-body-motion#29]
* Rigid Body Motion version: 0.7.0
* Python version: 3.8
* matplotlib version: 3.5.1
* Operating System: Linux

### Description

Running the jupyter notebook version of the documentation at https://rigid-body-motion.readthedocs.io/en/latest/reference_frames.html, got this error when plotting the referential:

```
fig = plt.figure()
ax = fig.add_subplot(111, projection="3d")
rbm.plot.reference_frame(rf_world, ax=ax)
rbm.plot.reference_frame(rf_observer, rf_world, ax=ax)
fig.tight_layout()

```

this returned (showing only the end of the error message):

```
~/.virtualenvs/mechanics/lib/python3.8/site-packages/mpl_toolkits/mplot3d/axes3d.py in do_3d_projection(artist)
    424                 """
    425                 try:
--> 426                     signature = inspect.signature(artist.do_3d_projection)
    427                     signature.bind()
    428                 # ValueError if `inspect.signature` cannot provide a signature

AttributeError: 'Arrow3D' object has no attribute 'do_3d_projection'
```

### Solution

Add the following to the Arrow3D class in `plotting.py`:

```
    def do_3d_projection(self, renderer=None):
        xs3d, ys3d, zs3d = self._verts3d
        xs, ys, zs = proj3d.proj_transform(xs3d, ys3d, zs3d, self.axes.M)
        self.set_positions((xs[0],ys[0]),(xs[1],ys[1]))

        return np.min(zs)
```

Reference: https://github.com/matplotlib/matplotlib/issues/21688

**Comment by phausamann** [comment]
Thanks for the report (and the fix) @diplodocuslongus . I've included the fix in the new release [0.9.0](https://github.com/phausamann/rigid-body-motion/releases/tag/v0.9.0).

**Comment by LYF-hahaha** [comment]
Thanks for share, I've fixed problem with adding contents above into Arrow3D class
